src/app/devices/pnp/components/deviceCommands/deviceCommandsPerInterfacePerCommand.tsx

```diff
@@ -36,6 +37,8 @@ export interface CommandSchema {
 export const DeviceCommandsPerInterfacePerCommand: React.FC<DeviceCommandDataProps & DeviceCommandDispatchProps> = (props: DeviceCommandDataProps & DeviceCommandDispatchProps) => {
     const { t } = useTranslation();
     const { collapsed, deviceId, moduleId, componentName, commandModelDefinition, parsedSchema, handleCollapseToggle, invokeCommand  } = props;
+    const [ confirmingSend, setConfirmingSend ] = React.useState<boolean>(false);
```

Comment:
recommand that we rename the variables to reflect what's happening.
const [ showConfirmationDialog, setShowConfirmationDialog ] = React.useState<boolean>(false);
const [ comandDataConfirmed, setComandDataConfirmed ] = React.useState({});